bazel/repositories_extra.bzl

```diff
@@ -8,7 +8,7 @@ def _python_minor_version(python_version):
     return "_".join(python_version.split(".")[:-1])
 
 # Python version for `rules_python`
-PYTHON_VERSION = "3.11.3"
+PYTHON_VERSION = "3.11.4"
```

Comment:
probably better to update this to 3.11.9

Reply:
Rules python version 0.26(https://github.com/envoyproxy/envoy-openssl/blob/6629924626298fdfc1c0fd52ae596e4f54a8c037/bazel/repository_locations.bzl#L936)  have support until 3.11.6 for 3.11 version (https://github.com/bazelbuild/rules_python/blob/fe33a4582c37499f3caeb49a07a78fc7948a8949/python/versions.bzl#L346) and next we have is 3.12.0. 

Minimum 3.11.4 is required to support s390x.

Reply:
ah k - this is on release/v1.28 branch so lower version for `rules_python` - i would update to latest in 3.11.x - 3.12 fails when i have tested iirc due to missing pre-compiled bins for other python deps